packages/explat-client/src/create-explat-client.ts

```diff
@@ -77,26 +83,40 @@ export function createExPlatClient( config: Config ): ExPlatClient {
 	 */
 	const createWrappedExperimentAssignmentFetchAndStore = ( experimentName: string ) =>
 		Timing.asyncOneAtATime( async () => {
+			let startTime: number | null = null;
+			try {
+				startTime = performance.now();
+			} catch ( e ) {}
 			const fetchedExperimentAssignment = await Request.fetchExperimentAssignment(
 				config,
 				experimentName
 			);
 			experimentAssignmentStore.store( fetchedExperimentAssignment );
+			let timeSinceStartMs = '';
+			try {
+				timeSinceStartMs =
+					startTime && performance.now() ? '' + ( performance.now() - startTime ) : '';
+			} catch ( e ) {}
```

Comment:
What kind of error is the `try`/`catch` block guarding against? `performance.now` is available in all supported browsers and always returns a valid value. We're using it at several other important places like Signup.

Checking if `performance.now()` is truthy also seems redundant. It always returns a positive number.

Reply:
Yeah this is just a temporary PR likely applied for today only and wasn't too familiar with performance.now and what browsers we support. I'll update it if it will always be there though